optax/_src/alias.py

```diff
@@ -66,6 +68,75 @@ def adabelief(
   )
 
 
+def adafactor(
+    learning_rate: Optional[float] = None,
+    min_dim_size_to_factor: int = 128,
+    decay_rate: float = 0.8,
+    decay_offset: int = 0,
+    multiply_by_parameter_scale: float = True,
+    clipping_threshold: Optional[float] = 1.0,
+    momentum: Optional[float] = None,
+    dtype_momentum: Any = jnp.float32,
+    weight_decay_rate: Optional[float] = None,
+    eps: float = 1e-30,
+    factored: bool = True,
+    ) -> base.GradientTransformation:
+  """The Adafactor optimiser.
+
+  Adafactor is an adaptive learning rate optimiser that focuses on fast
+  training of large scale neural networks. It saves memory by using a factored
+  estimate of the second order moments used to scale gradients.
+
+  References:
+    [Zhuang et al, 2020](https://arxiv.org/abs/2010.07468)
```

Comment:
Small suggestion:

```suggestion
    Zhuang et al, 2020: https://arxiv.org/abs/2010.07468
```

because of RTS rendering of Markdown-style links (see https://github.com/deepmind/optax/issues/113)

Reply:
Done